Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -415,14 +415,9 @@ public final class DiscoveryPageViewModel: DiscoveryPageViewModelType, Discovery
 
     // MARK: - Tracking
 
-    requestFirstPageWith
-      .observeValues { params in
-        let optimizelyProps = optimizelyProperties() ?? [:]
-
-        AppEnvironment.current.ksrAnalytics.trackDiscovery(
-          params: params,
-          optimizelyProperties: optimizelyProps
-        )
+    Signal.combineLatest(requestFirstPageWith, self.viewWillAppearProperty.signal)
```

Comment:
How do you feel about something like this:
`    requestFirstPageWith.signal
      .takeWhen(self.viewWillAppearProperty.signal)
      .observeValues { params in
        AppEnvironment.current.ksrAnalytics.trackDiscovery(params: params)
      }`

Removes the _ in the observeValues block. Test it and see if it gives the same desired effect.